aytests/bug-870820.xml

```diff
@@ -0,0 +1,392 @@
+	  <partition_nr config:type="integer">3</partition_nr>
+        </partition>
+        <partition>
+          <mount>/</mount>
+	  <label>SLES</label>
+          <filesystem config:type="symbol">btrfs</filesystem>
+          <format config:type="boolean">true</format>
+          <size>30%</size>
+	  <partition_nr config:type="integer">4</partition_nr>
+        </partition>
+        <partition>
+          <filesystem config:type="symbol">ext3</filesystem>
+          <label>ABUILD</label>
+          <mount>/abuild</mount>
+          <size>max</size>
+          <format config:type="boolean">true</format>
+	  <partition_nr config:type="integer">5</partition_nr>
+        </partition>
+      </partitions>
+    </drive>
+  </partitioning>
+  <report>
+    <errors>
+      <log config:type="boolean">true</log>
+      <show config:type="boolean">true</show>
+      <timeout config:type="integer">700</timeout>
+    </errors>
+    <messages>
+      <log config:type="boolean">true</log>
+      <show config:type="boolean">true</show>
+      <timeout config:type="integer">700</timeout>
+    </messages>
+    <warnings>
+      <log config:type="boolean">true</log>
+      <show config:type="boolean">true</show>
+      <timeout config:type="integer">700</timeout>
+    </warnings>
+    <yesno_messages>
+      <log config:type="boolean">true</log>
+      <show config:type="boolean">true</show>
+      <timeout config:type="integer">700</timeout>
+    </yesno_messages>
+  </report>
+  <!-- }}} -->
+  <!-- Runlevel {{{ -->
+  <runlevel>
+    <default>3</default>
+    <services config:type="list">
+      <service>
+        <service_name>SuSEfirewall2</service_name>
+        <service_status>disable</service_status>
+      </service>
+      <service>
+        <service_name>nfs</service_name>
+        <service_start>3 5</service_start>
+      </service>
+      <service>
+        <service_name>atd</service_name>
+        <service_start>3 5</service_start>
+      </service>
+      <service>
+        <service_name>nscd</service_name>
+        <service_status>disable</service_status>
+      </service>
+      <service>
+        <service_name>smartd</service_name>
+        <service_status>disable</service_status>
+      </service>
+      <service>
+        <service_name>sshd</service_name>
+        <service_status>enable</service_status>
+   <!-- <service_start>3 5</service_start> -->
+      </service>
+    </services>
+  </runlevel>
+  <!-- }}} -->
+  <!-- Scripts {{{ -->
+  <scripts>
+    <init-scripts config:type="list">
+      <script>
+        <!-- retrieving the scripts from FTP source doesn't work since YaST seems to
+             retrieve the stuff before we have (again) network ... -->
+        <filename>02-install_scripts</filename>
+        <debug config:type="boolean">false</debug>
+        <location/>
+        <source><![CDATA[#!/bin/sh
+          mkdir /tmp/scripts
+          cd /tmp/scripts
+          wget ftp://music.arch.suse.de/scripts/after_install.sh
```

Comment:
Seems that part was used for some env configuration. I've tested it with openQA and it works fine: http://gershwin.arch.suse.de/tests/1233, so I've simply removed this part.